FreeRTOS/FreeRTOS
# Changed project AVR_Dx_IAR to use 'heap_1.c'.

AVR Dx IAR Project update

Description
-----------
Changed project AVR_Dx_IAR to use 'heap_1.c'.

Test Steps
-----------
Project can now compile because it references valid heap implementation (from MemMang).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
